tests/generic_views/test_base.py

```diff
@@ -286,6 +291,11 @@ def test_direct_instantiation(self):
         response = view.dispatch(self.rf.head('/'))
         self.assertEqual(response.status_code, 405)
 
+    def test_self_not_in_signature_of_view_generated_by_as_view(self):
```

Comment:
@abbasidaniyal We can remove this test now as we're already checking that `__wrapped__` is not set in the test above as that was the source of the problem.